docs/conditional.rst

```diff
@@ -25,114 +23,80 @@ Suppose that you have defined a `Thread` Model for the examples below.
         subject = UnicodeAttribute(range_key=True)
         views = NumberAttribute(default=0)
 
-
-AND vs. OR
-^^^^^^^^^^
-
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+.. _conditions:
+
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
+
+PynamoDB supports creating condition expressions from attributes using a simple syntax.
```

Comment:
Again, I would state that you are using the Python builtin-operators here.

Reply:
Do you have a suggested wording? Not all expressions use built-in operators, some are done via methods on the attribute class.